runtime/compiler/z/codegen/J9CodeGenerator.hpp

```diff
@@ -300,6 +300,40 @@ class OMR_EXTENSIBLE CodeGenerator : public J9::CodeGenerator
     *    per node.
     */
    int32_t getMinimumNumberOfNodesBetweenMonitorsForTLE() { return 45; }
+
+   /** \brief
+    *     Sets whether decimal overflow or fixed point overflow checks should be generated for instructions which
+    *     support such by-passes.
+    *
+    *  \note
+    *     This is applicable to z15 hardware accelerated vector packed decimal operations and is typically used to
+    *     control whether the ignore overflow mask (IOM) bit is set in vector packed decimal instructions.
+    */
+   void setIgnoreDecimalOverflowException(bool v)
+      {
+      ignoreDecimalOverflowException = v;
+      }
+
+   /** \brief
+    *     Gets whether decimal overflow or fixed point overflow checks should be generated for instructions which
+    *     support such by-passes.
+    *
+    *  \note
+    *     This is applicable to z15 hardware accelerated vector packed decimal operations and is typically used to
+    *     control whether the ignore overflow mask (IOM) bit is set in vector packed decimal instructions.
+    */
+   bool getIgnoreDecimalOverflowException()
+      {
+      return ignoreDecimalOverflowException;
+      }
+
+   private:
+
+   /** \brief
+    *     Determines whether decimal overflow or fixed point overflow checks should be generated for instructions which
+    *     support such by-passes.
+    */
+   bool ignoreDecimalOverflowException;
```

Comment:
Other classes use a preceding `_` for private fields (though arguably it isn't super consistent throughout the code base).  Just pointing it out though.

Reply:
Fixed in cbc0737